tools/config/config_to_c.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
```

Comment:
Do we still need this file?

Reply:
I dont know. That is not in the scope of this PR in my opinion.

Reply:
True, but still wondering

Reply:
I don't think for that one. Maybe for the parser in the other comment.